ProjectStructureAndRoles.md

```diff
@@ -6,11 +6,6 @@
 
 The CAMARA Project is organized primarily into Sub Projects. Each Sub Project is comprised of Project participants from multiple companies and organizations, with a common purpose of advancing the Sub Project with respect to a specific Service API topic, for example ‘quality on demand’ or ‘localization’. Our goal is to enable distributed decision making and code ownership. This will be done by providing focused forums for getting work done, making decisions, and onboarding new Contributors.
 
-Areas covered by Sub Projects may be vertically focused on particular Service API components or functions, cross-cutting/horizontal or spanning many/all functional areas of Service APIs. Some examples of each type are:
-
-- Vertical: Quality on Demand, Localization
-- Horizontal: Scalability, Architecture, Lifecycle, Service
-
 Each Sub Project is documented in a separate repository.
 
 Each Sub Project must have a README.MD file (with a description of the Sub Project), a CODEOWNERS and a MAINTAINERS.MD file. A Sub Project should have at least one Maintainer. Ideally a Sub Project is managed by two or more Maintainers, depending on the size and scope of the Sub Project. Here, the responsibilities must be clearly agreed upon between all of the Maintainers. This includes coordinating who is responsible for which issues and pull requests. The Contributors to a Sub Project are recorded by the mechanisms of GitHub in the Sub Project’s repository. Each Contributor, Codeowner and Maintainer should also be listed in the [PARTICIPANTS.MD](./PARTICIPANTS.MD) file in the Governance repository root directory. Each Sub Project should have a license file, a GOVERNANCE.MD file (pointing to the Governance repository) and subdirectories /documentation/API_documentation, /documentation/MeetingMinutes, /documentation/SupportingDocuments, /code/API_code and /code/API_definitions. Writing permission to a Sub Project only codeowners should have.
```

Comment:
Why "Ideally, a Sub Project is managed by two or more Maintainers". What about, as a general rule, having one single maintainer, and for those cases where the Sub Project gets bigger (and thus difficult to be managed by one single maintainer), open the door for additional maintainers?